h2o-core/src/main/java/water/rapids/ast/prims/reducers/AstCumu.java

```diff
@@ -37,25 +39,121 @@ public String str() {
   @Override
   public ValFrame apply(Env env, Env.StackHelp stk, AstRoot asts[]) {
     Frame f = stk.track(asts[1].exec(env)).getFrame();
+    AstRoot axisAR = asts[2];
+    for (Vec v:f.vecs()) {
+      if(v.isCategorical() || v.isString()) throw new IllegalArgumentException(
+              "Cumulative functions not applicable to enum or string values");
+    }
+    int axis = (int) axisAR.exec(env).getNum();
```

Comment:
Could you please document this new parameter: what does axis=0 or axis=1 mean? What if I pass axis=2?